exasol_script_languages_container_ci/lib/release.py

```diff
@@ -39,10 +41,12 @@ def release(ctx: click.Context,
 
     ci_build(ctx, flavor_path=flavor_path, rebuild=True, build_docker_repository="",
              commit_sha="", docker_user="", docker_password="")
-    ci_test(ctx, flavor_path=flavor_path)
+    ci_test(ctx, check_for_skip_test=False, flavor_path=flavor_path, git_access=git_access)
     if not is_dry_run:
         ci_push(ctx, flavor_path=flavor_path,
                 target_docker_repository=docker_release_repository, target_docker_tag_prefix="",
                 docker_user=docker_user, docker_password=docker_password)
+    else:
+        logging.info("Skipping push to docker release repository.")
```

Comment:
Add: info that we skip because of dry run

Reply:
Done